src/jsapi.rs

```diff
@@ -4490,14 +4490,14 @@ extern "C" {
                                                                                              scopeChain:
                                                                                                  *mut AutoObjectVector)
      -> *mut JSObject;
-    fn _Z25JS_BufferIsCompilableUnitP9JSContextN2JS6HandleIP8JSObjectEEPKcj(cx:
+    fn _Z25JS_BufferIsCompilableUnitP9JSContextN2JS6HandleIP8JSObjectEEPKcm(cx:
                                                                                 *mut JSContext,
                                                                             obj:
                                                                                 Handle<*mut JSObject>,
                                                                             utf8:
                                                                                 *const i8,
                                                                             length:
-                                                                                u32)
+                                                                                u64)
```

Comment:
Is `libc::size_t` different from `usize`? It would be nice to use Rust ints whenever possible.